core/vm/contracts.go

```diff
@@ -151,6 +151,7 @@ func init() {
 func ActivePrecompiles(rules params.Rules) []common.Address {
 	switch {
 	case rules.IsPrague:
+		// Note: this config should be correctly set depending on replay starting point.
 		return PrecompiledAddressesBerlin
```

Comment:
In our previous replay branch, we had to override this to be `PrecompiledAddressByzantium`. I think now the current configuration since Shanghai precompiles are the same as Berlin, correct?

Left a comment here to remember to keep updating this as we keep rebasing Verkle further down the forks. I can remove it though, since maybe from now forward the default branch and replay data will always match.